src/module.c

```diff
@@ -1376,12 +1400,15 @@ NAPI_METHOD(dcn_stop_threads) {
   if (dcn_context->imap_thread && dcn_context->smtp_thread) {
```

Comment:
i think here is a `&& dcn_context->mvbox_thread` missing

Reply:
Aye, I've seen it too, but shouldn't affect the race.